### DISCUSSION TITLE
Double mark and signs for the statuscolumn on wrapped line.

### DISCUSSION THREAD

**Discussion by lkhphuc** [LazyVim/LazyVim#1653] [Q&A] [answered]
I have removed nvim-ufo for fold and statuscolumn.nvim and creating my own compact version based on LazyVim's new statuscolumn code.

However for wrapped line, all the statuscolumn are currently repeated over all the virtual wrapped lines. This looks great for gitsigns but a bit annoying for marks, diagnostic and code actions sign. 

LazyVim's new statuscolumn vs the default 'statuscolumn='
![Screenshot 2023-10-10 at 17 16 46](https://github.com/LazyVim/LazyVim/assets/12573521/4a0c8d0d-ba35-4f58-9a88-df642619ab79)  ![Screenshot 2023-10-10 at 17 18 09](https://github.com/LazyVim/LazyVim/assets/12573521/14ccd10a-0982-4bf4-8fe9-1e7d2a850432)

Does anyone know how to keep this duplicate behaviour for gitsign but not duplicate for the rest?

**Comment by folke** [comment, marked as answer]
https://github.com/LazyVim/LazyVim/issues/1654

I'll look into it
